### PR TITLE
[4.0] Lastest Action Logs Module

### PR DESCRIPTION
**Pull request by brianteeman**
This simple PR updates the view of the module to J4 standards

### Before
image

### After
image

### CODE REVIEW

File: administrator/modules/mod_latestactions/tmpl/default.php [hunk 1]
```diff
@@ -9,27 +9,43 @@
...
-					<?php echo $item->message; ?>
-				</div>
-				<div class="span4">
-					<div class="small pull-right hasTooltip" title="<?php echo JHtml::_('tooltipText', 'JGLOBAL_FIELD_CREATED_LABEL'); ?>">
-						<span class="icon-calendar" aria-hidden="true"></span> <?php echo JHtml::_('date', $item->log_date, JText::_('DATE_FORMAT_LC5')); ?>
-					</div>
-				</div>
-			</div>
+		<tr>
+			<td>
```
Comment (Quy) [inline administrator/modules/mod_latestactions/tmpl/default.php:30]: Change to `<th scope="row">`?

**Comment by chmst** [comment]
I have tested this item :white_check_mark: successfully on 0638b4ac40b9a3265def30688f8e68261a2adb5c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24563">issues.joomla.org/tracker/joomla-cms/24563</a>.</sub>

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 0638b4ac40b9a3265def30688f8e68261a2adb5c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24563">issues.joomla.org/tracker/joomla-cms/24563</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24563">issues.joomla.org/tracker/joomla-cms/24563</a>.</sub>

**Comment by infograf768** [comment]
OK here.
Note: The helper should be namespaced and moved to a `Helper` folder. But I guess this will have to be done when com_actionlogs will be namespaced.

**Comment by brianteeman** [comment]
Yes I deliberately left that for another pr

**Comment by alikon** [comment]
some works on namespacing actionlogs has been already done in 3.9.x where x > 0 :smiley:

**Comment by infograf768** [comment]
If nobody on it, I am finishing a similar PR for mod_privacy_dashboard

**Comment by infograf768** [comment]
Please test https://github.com/joomla/joomla-cms/pull/24573

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**